processing/src/main/java/org/apache/druid/query/spec/SpecificSegmentQueryRunner.java

```diff
@@ -45,6 +46,9 @@
   private final QueryRunner<T> base;
   private final SpecificSegmentSpec specificSpec;
 
+  @VisibleForTesting
+  static final String CTX_SET_THREAD_NAME = "setProcessingThreadNames";
```

Comment:
Should we document this somewhere?
Maybe Query context.md ?

Reply:
Added documentation.